### PR TITLE
Footer: Include chanelrooh credits, link github

### PR DESCRIPTION
**Pull request by pjeweb**

### CODE REVIEW

File: apps/website/src/components/layout/footer/Footer.tsx [hunk 1]
```diff
@@ -12,22 +16,41 @@ export const Footer: React.FC = () => {
           <ul className="flex flex-wrap items-center justify-between">
             <li className="basis-full p-2 md:basis-1/3">
               <p>
-                Built by the Alveus.gg team and community,
-                <br />
+                Original design by{" "}
```
Comment (MattIPv4) [inline apps/website/src/components/layout/footer/Footer.tsx:19]:
```suggestion
                Original design by
                {" "}
```
Nit, so welcome to disagree, but I prefer putting these on a newline when I need them

File: apps/website/src/components/layout/footer/Footer.tsx [hunk 2]
```diff
@@ -12,22 +16,41 @@ export const Footer: React.FC = () => {
...
+                <Link
+                  href="https://chanelrooh.com"
+                  target="_blank"
+                  rel="noreferrer"
+                  className={footerLinkClasses}
+                >
+                  chanelrooh.com
```
Comment (MattIPv4) [inline apps/website/src/components/layout/footer/Footer.tsx:26]:
```suggestion
                  Chanelrooh
```
I think it'd be better to use their name?

File: apps/website/src/components/layout/footer/Footer.tsx [hunk 3]
```diff
@@ -12,22 +16,41 @@ export const Footer: React.FC = () => {
...
+                </Link>
+                ,<br />
```
Comment (MattIPv4) [inline apps/website/src/components/layout/footer/Footer.tsx:28]:
```suggestion
                {", "}
                <br />
```
Similar nitpick -- also adding a space, so the text copies correctly

File: apps/website/src/components/layout/footer/Footer.tsx [hunk 4]
```diff
@@ -12,22 +16,41 @@ export const Footer: React.FC = () => {
...
+                built by the{" "}
```
Comment (MattIPv4) [inline apps/website/src/components/layout/footer/Footer.tsx:29]:
```suggestion
                built by the
                {" "}
```
Similar nitpick

File: apps/website/src/components/layout/footer/Footer.tsx [hunk 5]
```diff
@@ -12,22 +16,41 @@ export const Footer: React.FC = () => {
...
+                  target="_blank"
+                  className={footerLinkClasses}
+                >
+                  <IconGithub
+                    size={16}
+                    className="mr-1 inline-block fill-current"
+                  />
+                  Alveus.gg team
+                </Link>{" "}
+                and community, <br />
```
Comment (MattIPv4) [inline apps/website/src/components/layout/footer/Footer.tsx:42]:
```suggestion
                and community
                {", "}
                <br />
```

File: apps/website/src/icons/IconGithub.tsx [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width={size}
+    height={size}
+    viewBox="0 0 98 96"
+    className={className}
+  >
+    <path
+      fillRule="evenodd"
+      clipRule="evenodd"
```
Comment (MattIPv4) [inline apps/website/src/icons/IconGithub.tsx:18]:
```suggestion
      fill="currentColor"
```
To be consistent with other icons, default to using the text color

File: apps/website/src/icons/IconGithub.tsx [hunk 7]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+type IconProps = {
+  size?: number;
+  className?: string;
+};
+
+const IconGithub: React.FC<IconProps> = ({ size = 24, className }) => (
```
Comment (MattIPv4) [inline apps/website/src/icons/IconGithub.tsx:8]:
```suggestion
const IconGitHub: React.FC<IconProps> = ({ size = 24, className }) => (
```
Super nitpick, GitHub's brand name has a capital `H` (would also apply to the file name)

File: apps/website/src/components/layout/footer/Footer.tsx (hunk 1 above)
Comment (pjeweb) [reply apps/website/src/components/layout/footer/Footer.tsx:19]: Yeah i do not like what prettier does here at all with the newlines, but I have it set to run on save. Not sure if I should disable it just for this ...

Comment (pjeweb) [reply apps/website/src/components/layout/footer/Footer.tsx:19]: I'll just put a /* prettier-ignore */ in front of the jsx block!

File: apps/website/src/components/layout/footer/Footer.tsx (hunk 2 above)
Comment (pjeweb) [reply apps/website/src/components/layout/footer/Footer.tsx:26]: Would ask her, but going with your suggestion first

File: apps/website/src/components/layout/footer/Footer.tsx (hunk 1 above)
Comment (MattIPv4) [reply apps/website/src/components/layout/footer/Footer.tsx:19]: (I'll be honest, TIL that prettier was being used here, I don't think I've been running it)

**Pull request merged**